<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>CLevelDlgUI</class>
  <widget class="QDialog" name="LevelDlgUI" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>238</width>
        <height>148</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>IsoLevel Value Dialog</string>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item>
        <widget class="QLabel" name="mpZMinLabel" >
          <property name="text" >
            <string>Z Min :</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="mpZMaxLabel" >
          <property name="text" >
            <string>Z Max :</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <widget class="QLabel" name="mpIsoLabel" >
              <property name="text" >
                <string>IsoLevel :</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="mpIsoLevelLineEdit" />
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <widget class="QPushButton" name="mpApplyButton" >
              <property name="text" >
                <string>Apply</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="mpCancelButton" >
              <property name="text" >
                <string>Cancel</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <connections>
    <connection>
      <sender>mpCancelButton</sender>
      <signal>clicked()</signal>
      <receiver>LevelDlgUI</receiver>
      <slot>reject()</slot>
    </connection>
    <connection>
      <sender>mpApplyButton</sender>
      <signal>clicked()</signal>
      <receiver>LevelDlgUI</receiver>
      <slot>applySlot()</slot>
    </connection>
  </connections>
</ui>
